cmake/projects/breakpad/hunter.cmake

```diff
@@ -12,11 +12,11 @@ hunter_add_version(
     PACKAGE_NAME
     breakpad
     VERSION
-    0.0.0-12ecff3-p1
+    0.0.0-12ecff3-p2
```

Comment:
Do not modify old entry, add new instead:
* https://docs.hunter.sh/en/latest/creating-new/update.html#update-package